bw2parameters/ecoinvent_units.txt

```diff
@@ -0,0 +1,17 @@
+# default "ton" is interpreted as "short ton" (2000 pounds)
+ton = 1000 kg
+
+# ecoinvent specific stuff
+unit = [unit]
+person = [person] = _ = guest
+night = [time]
+EUR2005 = [currency]
+kilo = 1000
+Becquerel = becquerel
+Sm3 = [Sm3]
```

Comment:
You can add normal cubic meter as well, see https://github.com/cmutel/flowmapper/blob/main/flowmapper/data/units.txt#L7C1-L7C34

Reply:
I think this requires some clarification. 

First off, "normal conditions" and "standard conditions" are very problematic terms. NIST defines "normal conditions" as 20°C and 1 atm. DIN defines it as 0°C and 1 atm. IUPAC defines "standard conditions" as 0°C and 1000 bar, however, before 1982 it defined it as 0°C and 1 atm (making it effectively equivalent to DIN normal conditions). Apart from these four, there are a range of industry-specific definitions, which may take different values altogether. Even if we assume naming consistency within all datasets of one database (I doubt it), assuming consistency between different databases is a leap of faith. This begs the question: do we really want to allow these conversions?

If we say "yes, we definitely want to convert between these", I see another problem. Taking the NIST definition for "normal conditions" and the current IUPAC definition for "standard conditions", I calculate the conversion factor as follows:

$V_n = V_s \cdot \dfrac{p_s}{p_n} \cdot \dfrac{T_n}{T_s} = V_s \cdot \dfrac{100000 Pa}{101325 Pa} \cdot \dfrac{298.15 K}{273.15 K} = 1.0592 \cdot V_s$

You propose a conversion factor of 1.0732 instead. Could you explain where it comes from?